simpleui-server/src/server-util.ts

```diff
@@ -19,6 +19,18 @@ export class ServerUtil {
     }
 
 
+    /**
+     * replaces all occurences of 'search' with 'replacement' in 'input'
+     * @param input
+     * @param search
+     * @param replace
+     * @returns
+     */
+    static replaceAllOccurences(input: string, search: string, replacement: string) {
+        return input.split(search).join(replacement);
+    }
```

Comment:
I don't think this works as expected:

I tried this:
`function replaceAllOccurences(input, search, replacement) {
        return input.split(search).join(replacement);
    }`
    
then    

`console.log(replaceAllOccurences("xxAxxBxxCxx", "xx", "----"))`  
`----A----B----C----`

but should be 

`--A--B--C--`